src/convertToObject.js

```diff
@@ -1,18 +1,35 @@
 'use strict';
-
+'eslint-disable no-console';
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles (see an example in [stylesString.js]
+ * (./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties (see an exampl in
+ * [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
-  // write your code here
+  const arrayOfStyles = sourceString
+    .split(';')
+    .map(style => style.trim())
+    .filter(style => style !== '');
```

Comment:
we go through array 2 times, let's remove this `map` and `filter` 

I think we can make check inside `reduce`. 

If our value is undefined we skip iteration